tools/tikv-control.md

```diff
@@ -99,6 +105,28 @@ In this command, the key is also the escaped form of raw key.
 
 To print the value of a key, use the `print` command.
 
+### Print some properties about region
+
+In order to record region state details, `TiKV` writes some statistic into SST files of regions.
+For take a look at these properties, run `tikv-ctl` with sub-command `region-properties`:
```

Comment:
(Put the two lines in a same line) -> In order to record **Region** state details, **TiKV** writes some **statistics** into **the** SST files of **Regions**. **To view** these properties, run `tikv-ctl` with sub-command `region-properties`: